pingcap/pd
# config: fix schedulers maybe lost after restarting

### What problem does this PR solve? 
When we enable dynamic config, if we use pd-ctl to add a scheduler, it won't update the config manager and it will be lost after restarting. It should be reviewed after #2153 is merged.

### What is changed and how it works?
This PR adds a callback function to update the config manager to solve the above problem.

### Check List 

Tests 

 - Manual test

1. start PD 
2. add a scheduler via pd-ctl
3. restart PD
4. use pd-ctl config show to check
